ow_plexil/src/plans/FaultHandlingPattern4.plp

```diff
@@ -0,0 +1,74 @@
+// The Notices and Disclaimers for Ocean Worlds Autonomy Testbed for Exploration
+// Research and Simulation can be found in README.md in the root directory of
+// this repository.
+
+// If no faults are occurring, this plan will pan 180 degrees, 
+// and then unstow and stow the arm.
+// If any fault is injected, the plan will pause and wait for resolution.
+// If a fault is injected during an operation, that operation will only
+// halt immediately if the fault injected is associated with it directly,
+// i.e. an antenna fault during panning or an arm fault during 
+// unstowing or stowing.
+// Otherwise, the pausing will visually become apparent when the next
+// operation does not begin due to it.
+
+#include "plan-interface.h"
+
+LibraryAction InitializeAntennaAndArm ();
+
+FaultHandlingPattern4:
+{
+
+  Boolean ImageDone = false;
+  Boolean UnstowDone = false;
+  Boolean StowDone = false;
+
+  LibraryCall InitializeAntennaAndArm ();
+
+  log_info ("Starting FaultHandlingPattern4 plan...");
+
+  Mission: 
+  {
+    
+    Image:
+    {
+      Repeat !ImageDone;
+      Start !Lookup(AntennaFault);
+      Skip ImageDone;
+
+      LibraryCall Pan (Degrees=180);
+
+      ImageDone = !Lookup(AntennaFault);
+
+    }
+
+    Unstow:
+    {
+      Repeat !UnstowDone;
+      Start !Lookup(ArmFault);
+      Skip UnstowDone;
+
+      LibraryCall Unstow(); 
+      
+      UnstowDone = !Lookup(ArmFault);
```

Comment:
A better test for done would checking that arm is in the unstowed position.   Since there is no readily available way to do this, you could stick with this approach for now.   The result will depend on exactly when an injected arm fault is cleared.  But in general it seems like this approach should work.

Reply:
Yep! I used this approach as you suggested several days ago using the ImageDone variable as an example.

Reply:
It would be good to put a comment here (and in Stow) explaining the limitation that it's not possible at present to check the success/failure of lander commands, or the current arm position, and that this check will incorrectly indicate the completion of the unstow if the fault is cleared before Unstow() finishes.

Reply:
I was just adding a comment along these lines, and I am confused about your final statement. Doesn't the arm fault have to be cleared in order for Unstow() to finish?